README.md
# www.bianjie.ai
Bianjie Website

docs/news.md
---
{
    layout: Layout,
    isDynamic: true,
    title: 企业动态,
    dynamicTitle: {
        title: 企业动态,
        subTitle: 了解企业最新动态、行业资讯及新闻报道
    },
    dynamicList: [
        {
            date: '10月11日',
            year: '2021',
            title: 亚洲数字银行与边界智能达成战略合作，打造支持自金融服务的新一代数字银行,
            description: 强强联合！亚洲数字银行和边界智能达成战略合作，采用边界的 IRITA （Cosmos 企业版）区块链架构，包括可支持全球跨链互联的 BSN 开放联盟链「文昌链 IRITA OPB」等，开发打造可支撑高效自金融服务并支持全球合法贸易的新一代数字银行服务基础设施。,
            imgName: https://www.bianjie.ai/resources/Bianjie/BJHOME-IMAGE/news/28.png,
            route: ,
            link: https://mp.weixin.qq.com/s/norXFKNcq4SZC_dzBnyNbg
        },
        {
            date: '9月30日',
            year: '2021',
            title: 海南国际文化艺术品交易中心采用 BSN「文昌链」NFT 技术打造全球化数字艺术品交易市场,
            description: 海南国际文化艺术品交易中心宣布依托 BSN 开放联盟链「文昌链」作为底层区块链基础设施打造数字艺术品市场，通过前沿区块链跨链技术，与国际主流区块链联通，实现全球优质数字艺术品和数字资产有序流通和交易。,
            imgName: https://www.bianjie.ai/resources/Bianjie/BJHOME-IMAGE/news/27.jpg,
            route: ,
            link: https://mp.weixin.qq.com/s/joDIE2BLGS_FRxMfHqxbsQ
        },
        {
            date: '9月28日',
            year: '2021',
            title: 边界智能在 BSN 全球合作伙伴大会：「文昌链 IRITA OPB」跨链 NFT 技术、可信跨境贸易、全球化艺术品等,
            description: 2021 年 9 月 26 日至 28 日，边界智能受邀参加区块链服务网络（BSN）全球合作伙伴大会，发表两个主题演讲并参与圆桌讨论。,
            imgName: https://www.bianjie.ai/resources/Bianjie/BJHOME-IMAGE/news/26.jpg,
            route: ,
            link: https://mp.weixin.qq.com/s/5O9KyiRt56IwBVbW6Bdy1g
        },
        {
            date: '9月13日',
            year: '2021',
            title: 跨链核心技术贡献者专访 | 边界智能是「区块链互联网皇冠上的宝石」,
            description: Cosmos 区块链互联网博客发表英文专访，向全球介绍了边界智能在 NFT 与跨链技术研发、企业级应用、跨生态桥接等方面的创新和贡献。,
            imgName: https://www.bianjie.ai/resources/Bianjie/BJHOME-IMAGE/news/25.jpg,
            route: ,
            link: https://mp.weixin.qq.com/s/n8rmKI73AEbMf8ZoARMMnQ
        },
        {
            date: '9月6日',
            year: '2021',
            title: 丝路之舟与边界智能达成战略合作，共建联合国贸发会新型数字贸易基础设施,
            description: 中国丝路集团全资子公司丝路之舟与边界智能联合宣布，就「基于区块链及大数据隐私保护技术的新一代数字贸易基础设施」达成战略合作，支持打造联合国贸发会的新型数字贸易基础设施平台 DODR。,
            imgName: https://www.bianjie.ai/resources/Bianjie/BJHOME-IMAGE/news/24.jpg,
            route: ,
            link: https://mp.weixin.qq.com/s/DZrPLe3J1hyeR1j_xGDOJQ
        },
        {
            date: '8月23日',
            year: '2021',
            title: 边界智能与搜云科技展开深度技术合作，助推中国传统优秀艺术品的全球可信跨链流转,
            description: 边界智能与搜云科技达成深度技术合作，结合区块链跨链 NFT 技术和传统艺术品，基于边界智能打造的 BSN 开放联盟链「文昌链 IRITA OPB」，为传统艺术品全球可信跨链流转提供技术支撑，助推中国优秀传统艺术全球化。,
            imgName: https://www.bianjie.ai/resources/Bianjie/BJHOME-IMAGE/news/23.jpg,
            route: ,
            link: https://mp.weixin.qq.com/s/xLnifazaHIB7QNIUuXVolw
        },
        {
            date: '8月4日',
            year: '2021',
            title: 「文昌链 IRITA OPB」运用前沿 NFT 技术，助力 IP.PUB 推动中国传统艺术可信全球贸易,
            description: 边界智能基于自主研发的 BSN 开放联盟链「文昌链 IRITA OPB」及其 NFT 技术，助力中国民族文化数字文库 IP.PUB 支撑可信全球贸易，助推中国优秀传统艺术全球化。,
            imgName: https://www.bianjie.ai/resources/Bianjie/BJHOME-IMAGE/news/22.jpg,
            route: ,
            link: https://mp.weixin.qq.com/s/_AnxWMA9qQD2Qwh0fGfSGQ
        },
        {
            date: '7月26日',
            year: '2021',
            title: 边界智能荣登 2021 世界区块链大会「区块链名企 30」榜单,
            description: 7 月 24 日，2021 世界区块链大会在杭州召开，边界智能作为区块链和大数据领域的技术服务专家，凭借前沿的技术创新研发能力、丰富的产品和应用落地经验，边界智能荣登区块链「名企」30 榜单。,
            imgName: https://www.bianjie.ai/resources/Bianjie/BJHOME-IMAGE/news/21.jpg,
            route: ,
            link: https://mp.weixin.qq.com/s/3j2bAHGXtH5RiCT4s41WVw
        },
        {
            date: '7月14日',
            year: '2021',
            title: 边界智能成为 DIDA 分布式数字身份产业联盟成员单位,
            description: 7 月 11 日，分布式数字身份（苏州）高峰论坛暨分布式数字身份产业联盟 2021 年度会议及 DIDA 第一届理事会第二次理事会议召开，本次理事会议审议并表决通过，边界智能与其他 5 家企业成为联盟新成员单位。,
            imgName: https://www.bianjie.ai/resources/Bianjie/BJHOME-IMAGE/news/20.jpg,
            route: ,
            link: https://mp.weixin.qq.com/s/x1MQ7e-XfXa51tGwIZhLGw
        },
        {
            date: '7月13日',
            year: '2021',
            title: 边界智能 IRITA 通过中国信通院第五批可信区块链评测,
            description: 7 月 13 日，2021 可信区块链生态大会在北京召开，宣布了 2021 第五批可信区块链评测结果并颁发了证书，边界智能的创新技术产品再获认可，凭借 IRITA 底层链平台系统顺利通过中国信通院第五批可信区块链评测。,
            imgName: https://www.bianjie.ai/resources/Bianjie/BJHOME-IMAGE/news/19.jpg,
            route: ,
            link: https://mp.weixin.qq.com/s/e9ZOZ82coE-HVvYHJsa13A
        },
        {
            date: '7月13日',
            year: '2021',
            title: 边界智能荣登「2021 全球区块链企业创新 50 强」榜单,
            description: 7 月 10 日，2021 世界人工智能大会（WAIC）区块链论坛顺利举行，边界智能荣登「2021 全球区块链企业创新 50 强」榜单。,
            imgName: https://www.bianjie.ai/resources/Bianjie/BJHOME-IMAGE/news/18.jpg,
            route: ,
            link: https://mp.weixin.qq.com/s/u5o0iSbiIl3_WsimzKsx9w
        },
        {
            date: '7月9日',
            year: '2021',
            title: 边界智能赢得跨链基金会技术研发合同，作为核心技术贡献者受委托推进跨链开源技术研发,
            description: 边界智能与跨链基金会（Interchain Foundation，简称 ICF）签订了核心贡献者开发服务协议，本次签约更加明确了边界智能在跨链开源生态中担任的核心贡献团队角色。,
            imgName: https://www.bianjie.ai/resources/Bianjie/BJHOME-IMAGE/news/17.jpg,
            route: ,
            link: https://mp.weixin.qq.com/s/KDqyzo_YEcXi6Xq9ZKZQtQ
        },
        {
            date: '7月5日',
            year: '2021',
            title: 边界智能在南京创新周：IRITA、分布式商业应用、NFT 支持绿色金融等,
            description: 边界智能出席 2021 南京创新周紫金山创新大会，受邀参加第三届中国区块链技术产业发展峰会、扬子江新金融数字经济研讨会、 江北新区区块链技术应用大会暨万向区块链股份公司落地签约仪式等多场区块链技术会议与论坛，并发表多场主题演讲和圆桌分享。,
            imgName: https://www.bianjie.ai/resources/Bianjie/BJHOME-IMAGE/news/16.jpg,
            route: ,
            link: https://mp.weixin.qq.com/s/v7RVJrbscsDPqvlV0Zjxkg
        },
        {
            date: '6月29日',
            year: '2021',
            title: 解放日报报道：边界智能用前沿技术助力「绿色金融」,
            description: 解放日报就边界智能运用区块链 NFT 技术、大数据隐私保护技术等前沿科技助力绿色金融等方面的创新工作进行了采访与专题报道。,

src/assets/lang/cn.js
export const message = {
    "index": {
        "logo": [
            {
                "src": '../public/banner_01.png',
                "active": true
            },
            {
                "src": '../public/index_logo_cn.png',
                "active": false
            },
            {
                "src": '../public/banner_01.png',
                "active": true
            },
            {
                "src": '../public/index_logo_cn.png',
                "active": false
            },
        ],
        "title": [
            {"txt": "关于我们", "href": "/#about"},
            {"txt": "研发技术", "href": "/#core"},
            {"txt": "产品中心", "href": "/#product"},
            {"txt": "公司动态", "href": "/#dynamic"},
            {"txt": "合作伙伴", "href": "/#partner"},
        ]
    },
    "about": {
        "img": '../public/about.png',
        "title": "关于我们",
        "txt": [
            '上海边界智能科技有限公司（Bianjie.AI）创立于2016年，是一家专注于分布式智能处理和区块链技术开发的高科技公司。自创立以来，边界智能以人工智能，大数据分析和区块链跨链技术创新为核心，开发了IRISnet跨链服务网络、分布式大数据分析交换网络（BEAN）、智能医药知识库等核心产品，实现以保护数据隐私、尊重数据拥有者权利为前提的加密智能分析服务，并与复星星康链等合作伙伴一起在医疗健康大数据领域开展系列创新实践。'+
            '<br/><div style="text-indent: 34px">边界智能荣获第七届中国创新创业大赛成长组32强，中国首个面向以健康医疗大数据产业为核心的创新大赛 – CDHIC天使之星冠军等多个创新奖项。同时，边界智能也是全球著名跨链项目 COSMOS的核心中国开发者。</div>',
        ]
    },
    "core": {
        "title": "研发技术",
        "secondaryTitle": "跨链服务",
        "service": "支持异构网络间资产数字化流动，以及数据和复杂计算的可信跨链调用。",
        "technologyTitle": "分布式智能技术",
        "technology": "运用人工智能与区块链技术，保护数据隐私、尊重数据所有权，并支持数据依靠智能算法安全分布式地使用。"
    },
    "product": {
        "title": "产品中心",
        "txt": [
            {
                "title": "IRISnet",
                "bgImg": "../public/base_03.jpg",
                "img": "../public/base_03_03.png",
                "txt": "<a style='color: #72ACE3' href='https://www.irisnet.org' target='_blank'> IRISnet </a>是边界智能作为核心开发团队与Cosmos/Tendermint的研发团队合作打造的区块链跨链网络，专注于为分布式应用提供基础设施和协议，也将是Cosmos网络生态中的第一个区域性枢纽(Hub)。" +
                "<div><a style='color: #72ACE3' href='https://www.irisnet.org' target='_blank'> IRISnet </a>将面向服务的基础设施融入到Cosmos网络中，支持对包括公链、联盟链以及现有传统商业系统的集成从而实现互联互通。通过对Cosmos网络的跨链协议进行创新扩展，<a style='color: #72ACE3' href='https://www.irisnet.org' target='_blank'> IRISnet </a>允许数据及复杂计算跨异构网络被调用。" +
                "</div><div>就像忠实地在人间和天堂传递信息的希腊彩虹女神Iris，<a style='color: #72ACE3' href='https://www.irisnet.org' target='_blank'> IRISnet </a>的目标是成为链接数字经济和实体经济，支持构建复杂分布式商业应用的下一代公链。</div>"

            },
            {
                "title": "BEAN",
                "title1": "<span >B</span>lockchain <span>E</span>dge <span >A</span>nalytics <span >N</span>etwork",
                "bgImg": "../public/base_01.jpg",
                "img": "../public/base_01_01.png",
                "txt": "<div>区块链智能信息边缘分析网络BEAN，是基于 <a style='color: #72ACE3' href='https://www.irisnet.org' target='_blank'>IRISnet</a> 技术为大数据分析打造的分布式智能信息分析和数据交换网络，通过AI和区块链技术释放数据价值，支持数据可信分析和交换。</div>" +
                "<div>BEAN通过IRIS Hub和其他区块链网络交互，利用传统分布式人工智能研究中模型并行性和SOA设计模式，将区块链技术应用到分布式分析服务，实现与传统商业系统的链上链下通联，提供保护数据隐私、尊重数据拥有者权利的加密智能分析服务。" +
                "边界智能基于BEAN开发出支持医疗数据分析的应用，为医疗数据提供链上基于语义技术智能抽取，让非结构化医疗数据在保护隐私的前提下进行结构化处理，使得链上分布式业务协作时数据搜索、交换及使用变成可能。</div>"
            },
        ]
    },
    "dynamic": {
        "title": "公司动态",
        "btnTxt:": "查看更多",
        "txt": [
            [
                {
                    "year": '2021',
                    "day": "24",
                    "month": "June",
                    "title": "特等奖！边界智能获得 2021 年南京创新周 · 江北新区数字金融创新大赛特等奖",
                    "txt": "边界智能凭借《区块链 NFT 及大数据隐私保护技术支撑的绿色金融解决方案》获得南京创新周 · 江北新区数字金融创新大赛特等奖。",
                    "href": "https://mp.weixin.qq.com/s/s69uTkFOv7kyU14DgmhRBw"
                },
                {
                    "year": '2021',
                    "day": "22",
                    "month": "June",
                    "title": "上线可信身份服务，文昌链「IRITA OPB」推动互联网司法转型升级",
                    "txt": "文昌链「IRITA OPB」与壹证通展开合作，将识蛛可信身份服务运用到文昌链上，通过身份认证、数字证书等技术满足在线诉讼活动合法、真实、有效的需求。",
                    "href": "https://mp.weixin.qq.com/s/DAAcV9lYpGILkBRCJIZrlA"
                },
            ],
            [
                {
                    "year": '2021',
                    "day": "7",
                    "month": "June",
                    "title": "夺冠！边界智能荣获 2021 长三角（上海）区块链应用创新大赛成长组一等奖",
                    "txt": "边界智能的参赛方案《基于区块链的大数据共享解决方案》，凭借不俗的实力和亮眼的表现，获得2021 长三角（上海）区块链应用创新大赛成长组一等奖。",
                    "href": "https://mp.weixin.qq.com/s/9WRqTnDndmRIo0MVfzBUAw"
                },
                {
                    "year": '2021',
                    "day": "4",
                    "month": "June",
                    "title": "边界智能荣获「算力 20」榜单 2020 最具价值数据智能产品/解决方案奖",
                    "txt": "边界智能凭借在隐私计算和大数据领域的突出表现，荣获「算力 20」榜单 2020 最具价值数据智能产品/解决方案奖。",
                    "href": "https://mp.weixin.qq.com/s/gc_09Sg6F2yRNW8UC-r_LA"
                },
            ],
            [
                {
                    "year": '2021',
                    "day": "3",
                    "month": "June",
                    "title": "支持大数据持隐私保护及 NFT 的文昌链——边界智能在 2021 区块链服务网络（BSN）应用创新峰会分享",
                    "txt": "边界智能创始人曹恒受邀出席以「链上星城，链接未来」为主题的 2021 年区块链服务网络（BSN）应用创新峰会，并于大会做了《开放联盟链支持创新应用和产业发展——支持大数据隐私保护及 NFT 的 IRITA OPB：文昌链》的精彩分享。",
                    "href": "https://mp.weixin.qq.com/s/HeoA1ctfrk1waXy6_yriJw"
                },
                {
                    "year": '2021',
                    "day": "8",
                    "month": "May",
                    "title": "「智慧政务+区块链」都江堰市行政审批局智慧政务在 BSN 文昌链上线了",
                    "txt": "「政务服务底层区块链平台」应用项目是由都江堰区块链场景实验室的核心技术企业共同研发，边界智能承担了平台中「区块链电子证照系统」的搭建，该系统基于 BSN 首批开放联盟链「文昌链」开发，也是 BSN 开放联盟链「文昌链」部署的首个政务应用。",
                    "href": "https://mp.weixin.qq.com/s/P8xaUOcgl3m4BATDcL-vgQ"
                },
            ],
            [
                {
                    "year": '2021',
                    "day": "27",
                    "month": "Apr",
                    "title": "边界智能在 TBI 的分享：区块链互操作技术、打造新基建平台中的跨链服务、支持下一代分布式商业系统的开放联盟链",
                    "txt": "边界智能创始人曹恒受邀在可信区块链推进计划 2021 年第一次全会，在「互操作项目组」「电信应用工作组」「工业应用项目组、供应链协同应用项目组和 BaaS 工作组」 三个项目组研讨会中进行主题演讲。",
                    "href": "https://mp.weixin.qq.com/s/_tGFbVfxp9uQOGyBQPh2Cg"
                },
                {
                    "year": '2021',
                    "day": "27",
                    "month": "Mar",
                    "title": "边界智能成为中国中小企业协会产业区块链专委会常务理事单位",
                    "txt": "边界智能成为中国中小企业协会产业区块链专委会的 4 家常务理事单位之一，其联合创始人曹恒女士在第一次专委会会议上高票当选为常务理事会主席。",
                    "href": "https://mp.weixin.qq.com/s/q32jNkLaMnV8WGAiG6FcPw"
                }
            ],
            [
                {
                    "year": '2021',
                    "day": "22",
                    "month": "Mar",
                    "title": "西井科技与边界智能达成战略合作，共同推动 AI 商业化应用的隐私保护及数据确权",
                    "txt": "西井科技与边界智能，将围绕人工智能应用过程中的数据的确权与隐私保护领域达成战略合作，致力于推动数字化服务生态体系的建立，将物联网人工智能与隐私计算以及区块链跨链技术相结合，有效解决数据采集、流通、应用等各个环节的隐私保护及数据确权问题。",
                    "href": "https://mp.weixin.qq.com/s/QhrF4peejZ18hQDslCVZgg"
                },
                {
                    "year": '2021',
                    "day": "31",
                    "month": "Jan",
                    "title": "首批 BSN 开放联盟链「文昌链」上线",
                    "txt": "BSN 首批完成集成的开放联盟链「文昌链」已正式上线。「文昌链」是边界智能基于其自主可控联盟链产品 IRITA，改造了 IRIS Hub 和 Cosmos Hub 经过多年大规模实践验证过的前沿公有链底层技术，打造的支持多种智能合约的开放联盟链。",
                    "href": "https://mp.weixin.qq.com/s/F0QEBBG1uDVDWAEQDXMPBg"
                }
            ],
            [
                {
                    "year": '2021',
                    "day": "4",
                    "month": "Jan",
                    "title": "边界智能获 ISO9001 及 ISO27001 国际认证",
                    "txt": "边界智能完成了内部标准化管理体系建设的又一个里程碑：获得了 ISO9001 及 ISO27001 两项认证，这标志着边界智能在质量管理和信息安全管理方面达到了国际标准规范要求。",
                    "href": "https://mp.weixin.qq.com/s/dtRYOw2_LPJXEg_8ZdpN3g"
                },
                {
                    "year": '2020',
                    "day": "16",
                    "month": "Dec",
                    "title": "边界智能入选 2020 上海最具投资潜力 50 佳创业企业",
                    "txt": "边界智能凭借「支持跨链及大数据隐私保护的分布式商业系统 IRITA」顺利突出重围，成功获评「2020 上海最具投资潜力 50 佳创业企业」，是 50 佳获奖企业中为数不多的专注于区块链与大数据技术服务的高新技术企业。",
                    "href": "https://mp.weixin.qq.com/s/j6qgXPJJcbKvTxZHYLYsTg"
                }
            ],
            [
                {
                    "year": '2020',
                    "day": "14",
                    "month": "Dec",
                    "title": "万字庖解区块链跨链技术 | 金色财经采访实录",
                    "txt": "边界智能联合创始人曹恒接受金色财经采访，与红枣科技 CEO 何亦凡、分布科技创始人达鸿飞共同探讨区块链跨链技术与服务。",
                    "href": "https://mp.weixin.qq.com/s/5FPf1q5QFFLC3BYuJzjC4Q"
                },
                {
                    "year": '2020',
                    "day": "4",
                    "month": "Dec",
                    "title": "边界智能成为 IEEE 标准协会高级会员，参与区块链标准制定",
                    "txt": "边界智能正式成为 IEEE 标准协会高级会员，参与区块链跨链互操作等相关领域的标准制定。",
                    "href": "https://mp.weixin.qq.com/s/2aK7L4qCswbSDwTdFK-PaA"
                }
            ]
        ]
    },
    "partner": {
        "title": "合作伙伴",
        "img": [
            [
                {
                    "img": "../public/partner/wx.png",
                    "img_selected": "../public/partner/wx2.png",
                    "href": "http://www.wxblockchain.com/"
                },
                {
                    "img": "../public/partner/cosmos.png",
                    "img_selected": "../public/partner/cosmos2.png",
                    "href": "https://cosmos.network/"
                },
                {
                    "img": "../public/partner/hdf.png",
                    "img_selected": "../public/partner/hdf2.png",
                    "href": "http://www.haodf.com/"
                },
                {
                    "img": "../public/partner/nxylpt.png",
                    "img_selected": "../public/partner/nxyl2.png"
                },
            ],
            [
                
                {
                    "img": "../public/partner/bjdx.png",
                    "img_selected": "../public/partner/bjdx2.png",
                    "href": "http://www.bjmu.edu.cn/"
                },
                {
                    "img": "../public/partner/xwl_gray.png",
                    "img_selected": "../public/partner/xwl.png",
                    "href": "http://www.tonghaocapital.com/"
                },
                {
                    "img": "../public/partner/irisnet_gray.png",
                    "img_selected": "../public/partner/irisnet.png",
                    "href": "https://www.irisnet.org/"
                },
                {
                    "img": "../public/partner/xkl_gray.png",
                    "img_selected": "../public/partner/xkl.png",
                    "href": "http://healthcloud.link/"
                },
            ],
            [
                {
                    "img": "../public/partner/ptws_gray.png",
                    "img_selected": "../public/partner/ptws.png",
                    "href": "javascript:void(0)"
                }
            ]

        ]
    },
    'irisnetInfo':{
        "title":"IRIS Network",
        "irisnetService":'面向服务的区块链跨链网络',
        'irisnetTechnology':'新一代分布式商业生态的技术基础',
        "irisnetWebsiteUrl":'www.irisnet.org'
    },
    "validator":{
        "txt":[
            "是基于权益证明（PoS）Tendermint共识算法",
            "接入Cosmos区块链借助Hub将完成跨链价值转移",
            "边界智能作为Cosmos的Hub验证节点",
            "旨在维护Cosmos网络的安全性，促进生态发展！"
        ],
        'irisnetHref':"https://www.irisnet.org/",
        "btnText":"委托 ATOM",
        "btnHref":"/cosmos",
        "headerValidatorBtnText":"委托 ATOM"
    },
}
